index.fr.md

```diff
@@ -0,0 +1,147 @@
+ 
+## Interdépendances entre les composantes {#depend}
+ 
+Il y a des interdépendances cruciales entre les composantes&nbsp;; c’est-à-dire que les composantes doivent s’articuler correctement pour que le web soit accessible. Par exemple, pour les équivalents textuels aux images&nbsp;:
+ 
+-   les **spécifications techniques** traitent des équivalents textuels (par exemple, HTML définit l’attribut d’équivalent textuel (`alt`) pour l’élément image (`img`))&nbsp;;
+-   **les règles WAI** ([WCAG, ATAG, UAAG décrites ci-dessous](#guidelines)) définissent comment mettre en place les équivalents textuels dans la perspective de l’accessibilité pour les différentes composantes&nbsp;;
+-   les **créateurs** fournissent la formulation appropriée de l’équivalent textuel&nbsp;;
+-   les **outils de création** permettent, facilitent et promeuvent la fourniture d’un équivalent textuel dans une page web&nbsp;;
+-   les **outils d’évaluation** sont utilisés pour faciliter la vérification de la présence d’équivalents textuels&nbsp;;
+-   les **agents utilisateurs** fournissent des interfaces à destination des êtres humains et des machines pour l’équivalent textuel&nbsp;;
+-   les **technologies d’assistance** fournissent un accès aux êtres humains à l’équivalent textuel suivant différentes modalités&nbsp;;
+-   les **utilisateurs** savent comment obtenir l’équivalent textuel depuis leur agent utilisateur ou leur technologie d’assistance ou les deux, selon que de besoin.
+ 
+### Le cycle de mise en œuvre
+ 
+Quand les fonctionnalités d’accessibilité sont effectivement mises en œuvre dans une composante, les autres composantes sont plus susceptibles de les mettre en œuvre elles aussi.
+ 
+{% assign example_url = "/fundamentals/components/examples/#cycle" | relative_url %}
+![illustration du cycle de mise en œuvre, description détaillée à {{ example_url }}]({{ "/content-images/wai-components/cycle.fr.png" | relative_url }}){:longdesc="{{example_url}}"}
+ 
+- Quand **les navigateurs web, les lecteurs de média, les technologies d’assistance, et les autres agents utilisateurs** fournissent une fonctionnalité d’accessibilité, les utilisateurs sont plus susceptibles de l’exiger  et les créateurs sont plus susceptibles de l’implémenter dans leur **contenu**&nbsp;;
+- Quand les créateurs veulent implémenter une fonctionnalité d’accessibilité dans leur **contenu**, ils sont plus susceptibles d’exiger que leur **outil de création** en rende simple la mise en œuvre&nbsp;;
+- Quand **les outils de création** rendent simple la mise en œuvre d’une fonctionnalité, les créateurs sont plus susceptibles de la mettre en œuvre dans leur **contenu**&nbsp;;
+- Quand une fonctionnalité d’accessibilité est mise en œuvre dans  **la plupart des contenus**, les créateurs et les utilisateurs sont plus susceptibles d’exiger que les **agents utilisateurs** la fournissent.
```

Comment:
@3l3gant-cod3s What do you think of this alternative?

```suggestion
- Quand une fonctionnalité d’accessibilité est mise en œuvre dans  **la plupart des contenus**, les créateurs et les utilisateurs sont plus susceptibles d’exiger que les **agents utilisateurs** la supportent.
```

Reply:
anglicism ?
cf https://www.cnrtl.fr/definition/supporter

Reply:
@3l3gant-cod3s You are right, it is better to avoid anglicism when possible.

"prennent en charge" seems also a good option: https://vitrinelinguistique.oqlf.gouv.qc.ca/fiche-gdt/fiche/8390765/prendre-en-charge

Used for example in: https://developer.mozilla.org/fr/docs/Learn/CSS/CSS_layout/Supporting_Older_Browsers

It seems more subtle to me. Let me know if you think otherwise.

Reply:
@3l3gant-cod3s Can you do the change?